.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "db"]
   path = db
   url = https://github.com/unepwcmc/protectedplanet-db.git
-  branch = master
+  branch = feature/add-restricted-protected-areas-flag
```

Comment:
Could you please revert this back to the main branch?
This would only need to be changed locally

Reply:
Thanks @Levia I will change this too!